### ISSUE TITLE
Missing a couple IE11 grid vendor rules

### ISSUE THREAD

**Issue by basicdays** [frenic/csstype#58]
I'm not exactly sure how much control you have over missing css properties, but I ran into a few that are sometimes needed when working with IE11 and CSS Grids. I believe the following are some of the missing CSS properties:

- `-ms-grid-row`
- `-ms-grid-row-span`
- `-ms-grid-column`
- `-ms-grid-column-span`

I can probably dig up some more references if that'd be helpful.

**Comment by basicdays** [comment]
Found a couple more:

- `-ms-grid-column-align`
- `-ms-grid-row-align`

**Comment by frenic** [comment]
Thank you for raising this. My best guess would be to add them to [mdn/data](https://github.com/mdn/data/) rather than [mdn/browser-compat-data](https://github.com/mdn/browser-compat-data/) because it's an old spec. I created an issue to see what they have to say.

**Comment by pompomon** [comment]
I've found that those properties were recently added to [mdn/browser-conpat-data](https://github.com/mdn/browser-compat-data/pull/5530), please @frenic would that be OK from your point of view?

**Comment by frenic** [comment]
Yes, I noticed. But these changes will be ignored as they were added as separate properties and no syntaxes are specified at https://github.com/mdn/data/ for these properties. An alternative could be to add them as patches.

**Comment by pompomon** [comment]
I see, thanks for the clarification!
Please could you elaborate on adding them as patches? 
I'll gladly prepare a PR with the changes, but I'd need to understand what you are suggesting :)

**Comment by frenic** [comment]
Related https://github.com/mdn/data/pull/425
